Monitor notification rules (#28380)
DataDog/documentation: content/en/monitors/notify/_index.md

## Constructing effective titles and messages

This approach helps ensure your monitor titles and messages are clear, actionable, and tailored to your audience's needs.
- **Unique titles**: Add a unique title to your monitor (this is required). For multi alert monitors, some tags identifying your triggering scope are automatically inserted. You can use [tag variables][3] to enhance specificity.
- **Message field**: The message field supports standard [Markdown formatting][4] and [variables][5]. Use [conditional variables][6] to modulate the notification text sent to different contacts with [@notifications](#notifications).

{{% collapse-content title="Example monitor message" level="h4" expanded=false %}}
A common use-case for the monitor message is to include a step-by-step way to resolve the problem, for example:

```text
{{#is_alert}} <-- conditional variable

Steps to free up disk space on {{host.name}}: <-- tag variable

1. Remove unused packages
2. Clear APT cache
3. Uninstall unnecessary applications
4. Remove duplicate files

@slack-incident-response <-- channel to send notification

{{/is_alert}}

```

{{% /collapse-content %}}

## Notification recipients

> <div class="alert alert-info">Datadog recommends using <a href="/monitors/notify/notification_rules">Notification rules</a> to manage monitor notifications. You can also add notification recipients directly to individual monitors.</div>

Reviewer: This is great.

However I think we should add a quick paragraph explaining what notification rules are and how it works.
Otherwise in this page is all about @handles

Author: ```suggestion
Datadog recommends using Notification rules to manage monitor notifications. With notification rules you can automate which notification recipients are added to a monitor based on predefined sets of conditions. Create different rules to route monitor alerts based on the tags of the monitor notification so you don't have to manually set up recipients nor notification routing logic for each individual monitor. 
```

>  You can use an `@notification` to add a team member, integration, workflow, or case to your notification. As you type, Datadog auto-recommends existing options in a drop-down menu. Click an option to add it to your notification. Alternatively, click **@ Add Mention**, **Add Workflow**, or **Add Case**.

Author: ```suggestion
In both notification rules and individual monitors, you can use an `@notification` to add a team member, integration, workflow, or case to your notification. As you type, Datadog auto-recommends existing options in a drop-down menu. Click an option to add it to your notification. Alternatively, click **@ Add Mention**, **Add Workflow**, or **Add Case**.
```

{{% /collapse-content %}}

{{% collapse-content title="Integrations" level="h4" expanded=false %}}

{{% notifications-integrations %}}

{{% /collapse-content %}}

## Workflows
You can trigger a [workflow automation][8] or create a new workflow from a monitor. 

Before you add a workflow to a monitor, [add a monitor trigger to the workflow][9].

After you add the monitor trigger, [add an existing workflow to your monitor][10] or create a new workflow. To create a new workflow from the monitors page:

1. Click **Add Workflow**.
1. Click the **+** icon and select a Blueprint, or select **Start From Scratch**.
   {{< img src="/monitors/notifications/create-workflow.png" alt="Click the + button to add a new workflow" style="width:90%;">}}

For more information on building a workflow, see [Build workflows][11].

## Incidents 
Incidents can be automatically created from a monitor when the monitor transitions to an `alert`, `warn`, or `no data` status. Click on **Add Incident** and select an `@incident-` option. Admins can create `@incident-` options in [Incident Settings][12].

> Incidents created from a monitor will inherit its [field values][13] from the monitor's tags. To send automated notifications from incidents, add tags to the monitor so that created incidents match the criteria of [notification rules][14].

Reviewer: It's a bit confusing here when reading this as we say notification rules but this is referring to the incident one.

Author: I'm not familiar with this feature, how does this work with notification rules? If you add the `@incident` to notification rules will it work the same way?

And for the second paragraph, are incident notifications separate from monitor notifications or do they fall under the same umbrella? So if an incident created from a monitor is alerted, and the monitor notification rules match the incident tags, will the alerts carry the monitor notification rule recipients or the incident rule recipients?

## Further Reading

{{< partial name="whats-next/whats-next.html" >}}

[1]: /monitors/configuration
[2]: /service_management/case_management/create_case/#automatic-case-creation
[3]: /monitors/notify/variables/?tabs=is_alert#attribute-and-tag-variables
[4]: http://daringfireball.net/projects/markdown/syntax
[5]: /monitors/notify/variables/
[6]: /monitors/notify/variables/#conditional-variables
[7]: /account_management/teams/#send-notifications-to-a-specific-communication-channel
[8]: /service_management/workflows/
[9]: /service_management/workflows/trigger/#add-a-monitor-trigger-to-your-workflow
[10]: /service_management/workflows/trigger/#add-the-workflow-to-your-monitor
[11]: /service_management/workflows/build/
[12]: https://app.datadoghq.com/incidents/settings?section=global-settings
[13]: /service_management/incident_management/incident_settings/property_fields
[14]: /service_management/incident_management/incident_settings/notification_rules
[15]: /monitors/notify/variables/?tab=is_renotify#examples
[16]: /monitors/settings/#tag-policies
[17]: /account_management/teams/
[18]: /monitors/configuration/#set-alert-aggregation
[19]: /monitors/types
[20]: /monitors/configuration/

> [21]: /monitors/guide/recovery-thresholds/

Author: ```suggestion
[21]: /monitors/guide/recovery-thresholds/
[22]: /monitors/notify/notification_rules
```
